frontend/isoko/src/App.tsx

```diff
@@ -27,6 +33,20 @@ const App: React.FC = () => {
       region: 'us-west-2',
    });
 
+   const userPool = new CognitoUserPool({
+      UserPoolId: environment.cognitoUserPoolId,
+      ClientId: environment.cognitoAppClientId,
+   });
+
+   const [isOwner, setIsOwner] = useState(false);
```

Comment:
Is there a reason we need to store this in state? Once the profile is initialized with redux you should be able to just access it from the store like you do when you call `setIsOwner`